pkg/detectors/godaddy/v1/godaddy.go

```diff
@@ -0,0 +1,145 @@
+package godaddy
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+type GoDaddyEnv string
+
+var (
+	// ensure the scanner satisfies the interface at compile time.
+	_ detectors.Detector  = (*Scanner)(nil)
+	_ detectors.Versioner = (*Scanner)(nil)
+
+	defaultClient = common.SaneHttpClient()
+
+	// the key for the GoDaddy OTE environment is a 37-character alphanumeric string that may include underscores.
+	keyPattern = regexp.MustCompile(detectors.PrefixRegex([]string{"godaddy", "ote"}) + common.BuildRegex("a-zA-Z0-9", "_", 37))
+	// the secret for the GoDaddy OTE environment is a 22-character alphanumeric string.
+	secretPattern = regexp.MustCompile(detectors.PrefixRegex([]string{"godaddy", "ote"}) + common.BuildRegex("a-zA-Z0-9", "", 22))
+)
+
+// supported environments for godaddy
+var (
```

Comment:
questions: Is there a specific reason these can’t be `const`s? Also, I believe `OTE` is only used in the v1 detector, while `Prod` is used in the v2 detector. Could we move `Prod` to the v2 file? Lastly, do you think we could make them unexported? It doesn’t seem like they need to be visible outside this package, though I might be mistaken.

Reply:
Your suggestion is absolutely spot on. I guess I just did it for the style 😆